test/KnownCodesStorage.spec.ts

```diff
@@ -1,157 +1,157 @@
-                    .markFactoryDeps(true, [BYTECODE_HASH_2, INCORRECTLY_FORMATTED_HASH])
-            ).to.be.revertedWith('Incorrectly formatted bytecodeHash');
-        });
-
-        it('invalid length bytecode hash failed to call', async () => {
-            await expect(
-                knownCodesStorage
-                    .connect(bootloaderAccount)
-                    .markFactoryDeps(false, [INVALID_LENGTH_HASH, BYTECODE_HASH_3])
-            ).to.be.revertedWith('Code length in words must be odd');
-        });
-
-        it('successfuly marked', async () => {
-            await expect(
-                knownCodesStorage.connect(bootloaderAccount).markFactoryDeps(false, [BYTECODE_HASH_2, BYTECODE_HASH_3])
-            )
-                .to.emit(knownCodesStorage, 'MarkedAsKnown')
-                .withArgs(BYTECODE_HASH_2.toLowerCase(), false)
-                .emit(knownCodesStorage, 'MarkedAsKnown')
-                .withArgs(BYTECODE_HASH_3.toLowerCase(), false)
-                .not.emit(mockL1Messenger, 'MockBytecodeL1Published');
-            expect(await knownCodesStorage.getMarker(BYTECODE_HASH_2)).to.be.eq(1);
-            expect(await knownCodesStorage.getMarker(BYTECODE_HASH_3)).to.be.eq(1);
-        });
-
-        it('not marked second time', async () => {
-            await expect(
-                knownCodesStorage.connect(bootloaderAccount).markFactoryDeps(false, [BYTECODE_HASH_2, BYTECODE_HASH_3])
-            ).to.not.emit(knownCodesStorage, 'MarkedAsKnown');
-        });
-
-        it('sent to l1', async () => {
-            await expect(knownCodesStorage.connect(bootloaderAccount).markFactoryDeps(true, [BYTECODE_HASH_4]))
-                .to.emit(knownCodesStorage, 'MarkedAsKnown')
-                .withArgs(BYTECODE_HASH_4.toLowerCase(), true)
-                .emit(mockL1Messenger, 'MockBytecodeL1Published')
-                .withArgs(BYTECODE_HASH_4.toLowerCase());
-            expect(await knownCodesStorage.getMarker(BYTECODE_HASH_4)).to.be.eq(1);
-        });
-    });
-
-    describe('getMarker', function () {
-        it('not known', async () => {
-            expect(await knownCodesStorage.getMarker(INCORRECTLY_FORMATTED_HASH)).to.be.eq(0);
-        });
-
-        it('known', async () => {
-            expect(await knownCodesStorage.getMarker(BYTECODE_HASH_1)).to.be.eq(1);
-        });
-    });
-});
+// import { expect } from 'chai';
+// import { KnownCodesStorage, MockL1Messenger, MockL1Messenger__factory } from '../typechain-types';
+// import {
+//     BOOTLOADER_FORMAL_ADDRESS,
+//     EMPTY_STRING_KECCAK,
+//     L1_MESSENGER_SYSTEM_CONTRACT_ADDRESS,
+//     COMPRESSOR_CONTRACT_ADDRESS
+// } from './shared/constants';
+// import { Wallet } from 'zksync-web3';
+// import { getWallets, deployContract, loadArtifact, setCode, getCode } from './shared/utils';
+// import { network, ethers } from 'hardhat';
+
+// describe('KnownCodesStorage tests', function () {
```

Comment:
can you try 'xdescribe'? ( it is supposed to turn off the test, and print info that it was skipped)

Reply:
Of course, good idea! Updated everything accordingly.